server/routes/documents/documentRoutes.js

```diff
@@ -0,0 +1,30 @@
+const controller = require('../../controllers/documents');
+const daemon = require('../../middlewares');
+
+module.exports = (router) => {
+  // POST /documents
```

Comment:
Improve comment quality.
All you've specified I can understand from the code you are describing